javascript/operators/optional_chaining.json

```diff
@@ -78,4 +78,3 @@
     }
   }
 }
-  
```

Comment:
Can you undo this change? (add back new line at end of file)

Reply:
How peculiar my script was explicitly re-adding it to files which did not have it, perhaps my editor stripped it when I made manual changes. 

I'll update it. Nice spot.

Reply:
Thank you! I wonder if something is up this file. It is also a thing in another PR https://github.com/mdn/browser-compat-data/pull/5026#pullrequestreview-305531577 :eyes:

Reply:
Oh, I found out what it is: there’s two spaces at the end of the file that are getting removed, not the newline.  I’ll be merging that other PR to fix the issues!

Reply:
Merged the other PR now, so this is no longer an issue!